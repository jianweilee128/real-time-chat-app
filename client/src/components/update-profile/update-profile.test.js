import React from "react";
import { shallow } from "enzyme";

import UpdateProfile from "./update-profile.component";

describe("UpdateProfile component", () => {
  let wrapper;

  beforeEach(() => {
    const mockProps = {
      user: { name: "JohnDoe", id: "123" },
    };

    wrapper = shallow(<UpdateProfile.WrappedComponent {...mockProps} />);
  });

  it("should render UpdateProfile component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
