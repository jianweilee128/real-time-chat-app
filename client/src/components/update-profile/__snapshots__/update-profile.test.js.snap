// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UpdateProfile component should render UpdateProfile component 1`] = `ShallowWrapper {}`;
